corehq/motech/repeaters/views/repeat_records.py

```diff
@@ -240,9 +240,12 @@ def headers(self):
     def report_context(self):
         context = super().report_context
 
-        total = get_repeat_record_count(self.domain, self.repeater_id)
-        total_pending = get_pending_repeat_record_count(self.domain, self.repeater_id)
-        total_cancelled = get_cancelled_repeat_record_count(self.domain, self.repeater_id)
+        where = Q(domain=self.domain)
+        if self.repeater_id:
+            where &= Q(repeater_id=self.repeater_id)
+        total = SQLRepeatRecord.objects.filter(where).count()
+        total_pending = SQLRepeatRecord.objects.filter(where, state=State.Pending).count()  # include State.Fail?
```

Comment:
Does this question get answered in this PR?

Reply:
It is not answered in this PR. Failed records were not included previously, but given our definition of _pending_ I think it would make sense to include them. Just not sure if we should make that change as part of this PR? Input is welcome.

Reply:
Actually, if I do this at all I will do it in a separate PR. This is not the place to be changing how the system works.